pkg/storer/reserve.go

```diff
@@ -498,8 +498,8 @@ func (db *DB) SubscribeBin(ctx context.Context, bin uint8, start uint64) (<-chan
 }
 
 type NeighborhoodStat struct {
-	Address    swarm.Address
-	ChunkCount int
+	Address                 swarm.Address
```

Comment:
This field suggests that it is a swarm address, but it looks to me that it represents a neighborhood just using the swarm.Address type. I am questioning if both the field name and its type are correct. Maybe the field name can stay the same if the type would be of the specific neighborhood type. It looks to me that neighborhood would benefit from its own type, for example swarm.Neighborhood instead to use swarm.Address as a container that may mislead the code reader. There are technical similarities between two types but the semantics of them is different. Another option, field name could be Neighborhood for clarity.